[2025-02-04T01:38:54.987+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-04T01:38:55.003+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nasa_dag.create_data manual__2025-02-04T01:38:53.331183+00:00 [queued]>
[2025-02-04T01:38:55.011+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nasa_dag.create_data manual__2025-02-04T01:38:53.331183+00:00 [queued]>
[2025-02-04T01:38:55.012+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-04T01:38:55.024+0000] {taskinstance.py:2888} INFO - Executing <Task(MongoDBOperator): create_data> on 2025-02-04 01:38:53.331183+00:00
[2025-02-04T01:38:55.028+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=64) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-04T01:38:55.030+0000] {standard_task_runner.py:72} INFO - Started process 71 to run task
[2025-02-04T01:38:55.031+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'nasa_dag', 'create_data', 'manual__2025-02-04T01:38:53.331183+00:00', '--job-id', '278', '--raw', '--subdir', 'DAGS_FOLDER/nasa_dag.py', '--cfg-path', '/tmp/tmpwyoikbup']
[2025-02-04T01:38:55.032+0000] {standard_task_runner.py:105} INFO - Job 278: Subtask create_data
[2025-02-04T01:38:55.079+0000] {task_command.py:467} INFO - Running <TaskInstance: nasa_dag.create_data manual__2025-02-04T01:38:53.331183+00:00 [running]> on host fbf16bed15ba
[2025-02-04T01:38:55.153+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='nasa_dag' AIRFLOW_CTX_TASK_ID='create_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-04T01:38:53.331183+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-04T01:38:53.331183+00:00'
[2025-02-04T01:38:55.155+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-04T01:38:55.174+0000] {base.py:84} INFO - Retrieving connection 'mongo_default'
[2025-02-04T01:38:55.219+0000] {mongodb_file_operator.py:38} INFO - Collection etl_nasa already exists
[2025-02-04T01:38:55.229+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-04T01:38:55.230+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=nasa_dag, task_id=create_data, run_id=manual__2025-02-04T01:38:53.331183+00:00, execution_date=20250204T013853, start_date=20250204T013855, end_date=20250204T013855
[2025-02-04T01:38:55.247+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-04T01:38:55.271+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-04T01:38:55.274+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
